Week1

andreeaz19 opened the pull request:
Created an R package skeleton and added documentation to the R functions in the LGBTQsafety package.

andreeaz19 committed:
Added an R skeleton

Added an R skeleton and created two initial functions, sexuality_marriage_law()
 and gender_change_law().

andreeaz19 committed:
Added Documentation

Added documentation to the package functions, but this did not load into a new documentation file.

elena-sek commented on LGBTQsafety/R/Rpackage_functions.R:
Overall, the functions and their documentation are clear. Here are just some suggestions for improvement:

- You could load the PTNS_countries_LGBTQ.csv file directly from your github so that users don't have to download it and worry about the correct working directory, I did a similar thing for PIPS so maybe this example line of code helps:
`data <- read.csv("https://raw.githubusercontent.com/elena-sek/PIPS-Week4/main/CzechNames.csv", sep = ";")`

- You could name the function parameters more intuitively so instead of "a" you could use the parameter name "country"

elena-sek reviewed (commented):
The basic structure of the R package is there. The documentation contains all required elements and it clearly explains the individual functions. I included two suggestions for improvement in the comment of the functions file :)

jessiwitti commented:
The structure of the R package is there. I cannot find the automatically generated "man" folder with the documentation, but the documentation in your code is good. Thus, I can only agree with Elena :)

catherinegua reviewed (commented):
The code is clear and straightforward. The only thing I am confused by is why there is the assignment2_code file, where you build the package directly from your computer. 

I also agree that it would be more straightforward to have users be able to download your csv file directly from GitHub.

catherinegua reviewed (commented):
Nice that you added the tar file!

andreeaz19 committed:
Updated Documentation and New Functions

I have updated the documentation for the functions, including the arguments, so that the input will now be "country" rather than "a". I have added two new functions: read_data reads and downloads the dataset directly from the github repository to the global environment of the user, and discrimination_law provides information on whether laws prevent discrimination in different countries.
